lldb/source/Plugins/Process/gdb-remote/GDBRemoteCommunicationClient.cpp

```diff
@@ -1180,7 +1180,7 @@ bool GDBRemoteCommunicationClient::GetDefaultThreadId(lldb::tid_t &tid) {
 static void ParseOSType(llvm::StringRef value, std::string &os_name,
                         std::string &environment) {
   if (value.equals("iossimulator") || value.equals("tvossimulator") ||
-      value.equals("watchossimulator")) {
+      value.equals("watchossimulator") || value.equals("xrossimulator")) {
```

Comment:
I see in llvm/TargetParser/Triple.cpp it will accept either "xros" or "visionos" for Triple::XROS, I think this should accept "visionossimulator" too.